topbeat/beater/config.go

```diff
@@ -1,17 +1,48 @@
 package beater
 
+import (
+	"errors"
+	"time"
+)
+
+type ConfigSettings struct {
```

Comment:
We should standardise this naming across all beats. Suggestion:

This one we call `Config`, the one inside is `TopbeatConfig`. Same as we have in the cookiecutter: https://github.com/elastic/beat-generator/blob/master/%7B%7Bcookiecutter.beat%7D%7D/config/config.go

@urso I remember you once used even a nested notation, so the "empty object was not needed?